model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -635,6 +635,61 @@ message StandardCoders {
   }
 }
 
+// Experimental: A representation of a Beam Schema.
+message Schema {
+  enum TypeName {
+    BYTE = 0;
+    INT16 = 1;
+    INT32 = 2;
+    INT64 = 3;
+    DECIMAL = 4;
+    FLOAT = 5;
+    DOUBLE = 6;
+    STRING = 7;
+    DATETIME = 8;
```

Comment:
Adding schemas to portability needs a design doc and discussion on dev@. Did I miss that?